kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/server/AdbServer.kt

```diff
@@ -24,8 +24,29 @@ interface AdbServer {
      * @throws AdbServerException if a result status of any command in @param commands is Failed
      * @return list of answers of commands' execution
      */
+    @Deprecated("This method doesn't work for the commands with the complex arguments containing " +
+            "whitespaces (e.g. 'adb pull \"path/with whitespace/file\") and doesn't allow commands piping like" +
+            "adbServer.performCmd(\"bash\", listOf(\"-c\", \"adb shell dumpsys deviceidle | grep mForceIdle\"))" +
+            "which the AdbServer.performCmd(String, List<String>) does. " +
+            "For more details, please check out https://github.com/KasperskyLab/Kaspresso/blob/master/docs/Wiki/Executing_adb_commands.en.md",
+        ReplaceWith("adbServer.performCmd(*commands, emptyList())")
```

Comment:
Are you sure that it is correct replacement? You put an array (`*commands`) where `adbServer.performCmd` expects `command: String`.

Reply:
It's the best from the worst. If you leave just `commands` then IDE will replace `adbServer.performCmd("abc")` with `adbServer.performCmd(arrayOf("abc"), emptyList())` which will be highlighted as an error. I think that 90% of adbServer.perform... calls are with a signle command. For those cases, when there're several commands, then it will be replaced with `adbServer.performCmd("asd", "abc", emptyList())` which is ugly too, but less possible. It's a tradeoff. I don't want to leave ReplaceWith statement with a stub, because it will upset users by erasing their precious commands